configuration/cjdns.md

```diff
@@ -0,0 +1,229 @@
+```
+
+After reboot, the `cjdroute.conf` in `sys-cjdns` should have a different *privateKey* (and therefore different *publicKey* and *ipv6 address*) than the `cjdroute.conf` in the `cjdns-gw` TemplateVM.
+
+### 10. Verify Cjdns Proxy
+Now you should be able to `ping6` the Cjdns router in the ProxyVM from an AppVM. For example configure your `personal` VM to use the `sys-cjdns` VM as the network.
+
+Next, let's find out the cjdns address of your `sys-cjdns` VM. Cjdns addresses are IPv6 addresses which always begin with `fc`. Launch a  terminal in the `sys-cjdns` VM and type:
+```
+user@sys-cjdns:~$ sudo ifconfig | grep -B1 "inet6 fc"
+tun0: flags=4305<UP,POINTOPOINT,RUNNING,NOARP,MULTICAST>  mtu 1304
+        inet6 fc3e:c29a:0166:4c38:2243:b62f:3cb4:693f  prefixlen 8  scopeid 0x0<global>
+```
+In this example, you can see that Cjdns creates a `tun0` interface with the IPv6 address `fc3e:c29a:0166:4c38:2243:b62f:3cb4:693f`.
+
+Now launch a terminal in the AppVM you wired with `sys-cjdns` (e.g. the `personal` VM) and type:
+```
+[user@personal ~]$ ping6 -c3 fc3e:c29a:0166:4c38:2243:b62f:3cb4:693f
+PING fc3e:c29a:0166:4c38:2243:b62f:3cb4:693f(fc3e:c29a:0166:4c38:2243:b62f:3cb4:693f) 56 data bytes
+64 bytes from fc3e:c29a:0166:4c38:2243:b62f:3cb4:693f: icmp_seq=1 ttl=41 time=4.85 ms
+64 bytes from fc3e:c29a:0166:4c38:2243:b62f:3cb4:693f: icmp_seq=2 ttl=41 time=4.33 ms
+64 bytes from fc3e:c29a:0166:4c38:2243:b62f:3cb4:693f: icmp_seq=3 ttl=41 time=5.74 ms
+
+--- fc3e:c29a:0166:4c38:2243:b62f:3cb4:693f ping statistics ---
+3 packets transmitted, 3 received, 0% packet loss, time 2003ms
+rtt min/avg/max/mdev = 4.334/4.975/5.743/0.587 ms
+```
+You should see 3 successful pings, which means the new Qubes 4.0 IPv6 feature was correctly enabled for `sys-cjdns` and is working.
+
+### 11. Optional: Connect to other Cjdns peers in your local network
+If you have other Cjdns nodes running in your local network, you can add them to `/etc/cjdroute.conf` in the `sys-cjdns` ProxyVM now. As mentioned previously Cjdns zero-configuration doesn't work with Qubes, as `sys-cjdns` is behind a NAT as well and not connected directly to your local network.
+
+Just copy the example template from your other nodes `/etc/cjdroute.conf` that looks like:
+```javascript
+// my other peer: fc0d:378f:a61e:5223:6dd1:bd72:ad03:62fd
+"your.external.ip.goes.here:19264": {
+    "login": "default-login",
+    "password":"13yj3nk85myvt5r8pb8x5140n4x4cfg",
+    "publicKey":"tr459vnqfby0hrwc40ucnqbvg04snvfy1r3g9y61911t3b6hu910.k",
+    "peerName":"my-other-peer"
+}
+```
+Adjust *"your.external.ip.goes.here"* with the local IPv4 or IPv6 address of that other node and give it a meaningful *"peerName"*. Now add that snippet to `/etc/cjdroute.conf` of your `sys-cjdns` ProxyVM, precisely to the section *"interfaces" > "UDPInterface" > "connectTo"*. Be careful, this is a JSON-like syntax, so add a comma after the previous item when you add something. Else Cjdns won't start anymore. 
```

Comment:
It would be useful to include info how to get other parameters too (password, publicKey). Even if that's "check <link> how to get other parameters".